module/enrichers.mjs

```diff
@@ -25,6 +25,12 @@ export function registerEnrichers() {
       pattern: /\[\[\/language (\w+)]]/g,
       enricher: enrichLanguage
     },
+    {
+      id: "dnd5eAward",
```

Comment:
I would prefer just calling this `award` since I don't anticipate having a *different* enricher for crucible.